product_by_supplier/views/product_supplierinfo_view.xml

```diff
@@ -9,12 +9,12 @@
                     <field name="name" string="Supplier"/>
                     <field name="product_code" string="Supplier Product Code"/>
                     <field name="product_name" string="Supplier Product Code"/>
```

Comment:
Could you fix this copy paste error (unrelated to your port) and change the label for product_name to 'Supplier Product Name'?